packages/frontend-v2/utils/validation/noLeadOrTrailWhitespacePattern.ts

```diff
@@ -0,0 +1,10 @@
+import { ValidationRule } from "react-hook-form";
```

Comment:
shift this to common because it could be used by BE as well?

Reply:
truuuue, but for now this is coupled to react-hook-forms only. We can standardize and move to back end later, might just move the regex itself.